Makefile

```diff
@@ -68,7 +73,7 @@ dependencies:
 
 # release builds and installs release binaries.
 release: dependencies test-long
-	go install -a ./...
```

Comment:
REBUILD is already a dependency of both of them. I suppose it could be more explicit though.